README.md

```
@@ -82,6 +82,24 @@ Input Queries ──→  ├─── arXiv API (relevance sort)
                    └── Web UI (DAG + Tabs: Papers / Insights / Judge)
 ```
 
+### DailyPaper SSE 流式管线
+
+当启用 LLM 分析或 Judge 评分时，`/daily` 端点返回 SSE 流式响应，前端实时显示每个阶段的进度：
+
+```
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)